invokeai/app/invocations/latent.py

```diff
@@ -319,3 +319,54 @@ def invoke(self, context: InvocationContext) -> ImageOutput:
             return ImageOutput(
                 image=ImageField(image_type=image_type, image_name=image_name)
             )
+
+    # this should be refactored - duplicated code in diffusers.pipelines.stable_diffusion
+    @classmethod
+    def _decode_latents(self, vae:AutoencoderKL, latents:torch.Tensor)->Image:
```

Comment:
Is `AutoencoderKL` ours? Could we add this (and the below `_encode_latents`) to it?